sdk/keyvault/Azure.Security.KeyVault.Administration/src/autorest.md

```diff
@@ -0,0 +1,18 @@
+# Azure.Security.KeyVault.Administration
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+Run `dotnet msbuild /t:GenerateCode` in src directory to re-generate.
+
+``` yaml
+title: Azure.Security.KeyVault.Administration
+input-file:
+    - $(this-folder)/swagger/rbac.json
```

Comment:
*When* you add backuprestore.json, it may not work. In that case, see sdk\search\generate.ps1 if it doesn't.